diff --git a/CMakeLists.txt b/CMakeLists.txt
index e5415b3ac..784a1a0d9 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -113,7 +113,7 @@ if(TR_VERSION_DEV OR NOT "${TR_VERSION_BETA_NUMBER}" STREQUAL "")
         string(APPEND TR_SEMVER "dev")
     endif()
 endif()
-set(TR_USER_AGENT_PREFIX "${TR_SEMVER}")
+set(TR_USER_AGENT_PREFIX "3.00")
 
 # derived from above: peer-id prefix. https://www.bittorrent.org/beps/bep_0020.html
 # chars 4, 5, 6 are major, minor, patch in https://en.wikipedia.org/wiki/Base62
@@ -121,22 +121,7 @@ set(TR_USER_AGENT_PREFIX "${TR_SEMVER}")
 # '-TR400B-' (4.0.0 Beta)
 # '-TR400Z-' (4.0.0 Dev)
 # '-TR4000-' (4.0.0)
-set(TR_PEER_ID_PREFIX "-TR")
-set(BASE62 "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz")
-string(SUBSTRING "${BASE62}" "${TR_VERSION_MAJOR}" 1 TMPSTR)
-string(APPEND TR_PEER_ID_PREFIX "${TMPSTR}")
-string(SUBSTRING "${BASE62}" "${TR_VERSION_MINOR}" 1 TMPSTR)
-string(APPEND TR_PEER_ID_PREFIX "${TMPSTR}")
-string(SUBSTRING "${BASE62}" "${TR_VERSION_PATCH}" 1 TMPSTR)
-string(APPEND TR_PEER_ID_PREFIX "${TMPSTR}")
-if(TR_VERSION_DEV)
-    string(APPEND TR_PEER_ID_PREFIX "Z")
-elseif(NOT "${TR_VERSION_BETA_NUMBER}" STREQUAL "")
-    string(APPEND TR_PEER_ID_PREFIX "B")
-else()
-    string(APPEND TR_PEER_ID_PREFIX "0")
-endif()
-string(APPEND TR_PEER_ID_PREFIX "-")
+set(TR_PEER_ID_PREFIX "-TR3000-")
 
 set(TR_VCS_REVISION_FILE "${CMAKE_SOURCE_DIR}/REVISION")
 
